schema/auction_result.js

```diff
@@ -0,0 +1,80 @@
+import date from "./fields/date"
+import { amount } from "./fields/money"
+import { IDFields, NodeInterface } from "./object_identification"
+import { GraphQLFloat, GraphQLNonNull, GraphQLString, GraphQLObjectType } from "graphql"
+import { connectionDefinitions } from "graphql-relay"
+import { has } from "lodash"
+
+const AuctionResultType = new GraphQLObjectType({
+  name: "AuctionResult",
+  interfaces: [NodeInterface],
+  isTypeOf: obj => has(obj, "dimension_text") && has(obj, "organization"),
+  fields: () => ({
+    ...IDFields,
+    title: {
+      type: GraphQLString,
+    },
+    artist_id: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    date,
```

Comment:
does this get automatically typed?

Reply:
Yes, this uses the shorthand object syntax, so this is actually `date: date`.

And then that comes from the import at the top https://github.com/artsy/metaphysics/pull/821/files#diff-3bd45ce967b722bba06b950c617229dfR1 , which is our [date type](https://github.com/artsy/metaphysics/blob/975f2e3eafc98025f35bfb3dbd5d4e0daf90160c/schema/fields/date.js).

Reply:
:+1: